packages/transformers/image/src/ImageTransformer.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import {Transformer} from '@parcel/plugin';
+import sharp from 'sharp';
+
+const FORMATS = new Map([
+  ['heic', 'heif'],
+  ['heif', 'heif'],
+  ['jpeg', 'jpeg'],
+  ['jpg', 'jpeg'],
+  ['png', 'png'],
+  ['raw', 'raw'],
+  ['tiff', 'tiff'],
+  ['webp', 'webp'],
+]);
+
+export default (new Transformer({
+  async transform({asset}) {
+    asset.isIsolated = true;
+
+    let inputBuffer = await asset.getBuffer();
+    let width = asset.query.width ? parseInt(asset.query.width, 10) : null;
+    let height = asset.query.height ? parseInt(asset.query.height, 10) : null;
+    let quality = asset.query.quality
+      ? parseInt(asset.query.quality, 10)
+      : undefined;
+    let format = asset.query.as ? asset.query.as.toLowerCase().trim() : null;
+
+    let imagePipeline = sharp(inputBuffer);
```

Comment:
If none of the options are set, should we skip running sharp altogether? Maybe faster?

Reply:
But the output might be non minified

Reply:
Hmm it's not an optimizer... I guess we could check that option too. I was more thinking in dev it might be slow/unnecessary.

Reply:
Yeah... I can fix that in a follow-up PR if you want, sharp isn't always smaller anyway, I've had a couple cases in which it significantly enlarged output images...